examples/sql_extended_server_blob_auditing_policies/main.tf

```diff
@@ -0,0 +1,36 @@
+resource "azurerm_resource_group" "test" {
+  name     = "acctestRG"
+  location = "eastus"
+}
+
+resource "azurerm_sql_server" "test" {
+  name                         = "acctestsqlserver"
```

Comment:
The value maybe need to updated from acctestsqlserver to example-acctestsqlserver